lite/core/mir/weight_quantization_preprocess_pass.cc

```diff
@@ -22,24 +22,51 @@ namespace paddle {
 namespace lite {
 namespace mir {
 
+bool IsAbsMaxQuantizedOp(const OpInfo* op_info) {
```

Comment:
if not modify op_info, use `const OpInfo &op_info` instead.

Reply:
Done